docs/workforce-management-framework/topics/ref-demo-app.adoc

```diff
@@ -25,3 +25,30 @@ Login using the following credentials:
 |`max`
 |`123`
 |===
+
+[id=-'{context}-structure-for-the-mobile-and-portal-starter-applications']
+ifdef::upstream[]
+discrete
+endif::upstream[]
+== Structure for the Mobile and Portal Starter Applications
+
+The UI Starter Applications present a consistent directory structure that can serve as a sensible base for customizations.
+
+The following table highlights the function of the main folders:
+
+|==
```

Comment:
Issue with the table render 

![image](https://user-images.githubusercontent.com/16667688/31011525-8b8984c2-a506-11e7-9c99-755fb3165928.png)

Reply:
needs an extra =
```|===```